src/WmContentManager.php

```diff
@@ -203,33 +199,38 @@ public function getHostContainers($host)
     {
         $containers = $this->getContainers();
 
+        $trigger = $host->getEntityTypeId() . "|" . $host->bundle();
+
+        // Holder.
         $return = [];
 
+        // Go through all containers.
         foreach ($containers as $container) {
-            if (!$container->isHost($host)) {
-                continue;
-            }
+            // Look at the host bundles.
+            $typebundles = $container->host->host_typebundles;
 
-            $return[] = $container->getId();
+            // Go through all of them and if we are there give back that.
+            foreach ($typebundles as $typebundle) {
+                if ($typebundle == $trigger) {
+                    $return[] = $container;
+                }
+            }
         }
 
         return $return;
     }
-    
+
     /**
      * {@inheritdoc}
      */
     public function getLabel($entityType, $bundle)
     {
-        $labels = &drupal_static(__FUNCTION__);
```

Comment:
@brownrl I think the static function was there for a performance reason. Can you put it back in?